.eslintrc

```diff
@@ -0,0 +1,57 @@
+{
+  "parser"  : "babel-eslint",
+  "plugins": [
+    "import"
+  ],
+  "extends" : ["airbnb"],
+  "rules": {
+    // Soften some rules.
+    "comma-dangle": 0, // Nobody cares about commas.
+    "default-case": 0, // Required default case is nonsense.
+    "new-cap": [2, {"capIsNew": false, "newIsCap": true}], // For Record() etc.
+    "no-floating-decimal": 0, // .5 is just fine.
+    "no-shadow": 2, // Shadowing is a nice language feature, but it can kick
+    // eslint-plugin-import
+    "import/no-unresolved": [2, {"commonjs": true}],
+    "import/named": 2,
+    "import/default": 2,
+    "import/namespace": 2,
+    "import/export": 2,
+    // BB rules soften
+    "max-len": 0,
+    "key-spacing": [2, {"beforeColon": false, "afterColon": true, "mode": "minimum"}], // Enable use of nice block object creation.
+    "no-use-before-define": 0, // Enable to define styles after using them in component.
+    "react/jsx-no-bind": 0, // Enable arrow functions in Props definitions.
+    "react/prefer-stateless-function": 0, // Enable functions with state.
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+  },
+  "globals": {
```

Comment:
simplify it with https://www.npmjs.com/package/eslint-plugin-jest
and use env to handle browser/node globals
```json
    "env": {
        "browser": true,
        "node": true,
        "jest/globals": true,
    }